### PR TITLE
adds week 2 sgc minutes

### PR DESCRIPTION
**Pull request by finnhodgkin**

### CODE REVIEW

**Comment by finnhodgkin** [comment]
lol pr with no body and commit with no issue :see_no_evil: :speak_no_evil:

File: stop-go-continue/fac-11/cohort-week-2.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  + Jen(F9): ideally you would spend the whole project learning everything.
++ Reading pulls as a group
+  + Bart: Connected to my point in go. Every time there was a PR we went through it and it worked really well!
+
+### Go
+
++ Build more into romaniser code-along
+ + Jen: We hit a lot of stumbling blocks in our project that it would have been helpful to have primer on.
+ + Oli: So things like more examples of mocking data in tests?
+ + Jen: Yes.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-2.md]: Needs one more space for indentation (3 lines above)

File: stop-go-continue/fac-11/cohort-week-2.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
++ close the toilet door + turn light off
+ + R: it smells.
```
Comment (jsms90) [inline stop-go-continue/fac-11/cohort-week-2.md]: And here

**Comment by jsms90** [comment]
Indentation issues don't matter that much since this file needs to migrate to `london-programme` anyway

**Comment by finnhodgkin** [comment]
Eep sorry I completely forgot you asked for changes 😳

**Pull request merged**